packages/astro/test/astro-sync.test.js

```diff
@@ -62,6 +63,20 @@ const createFixture = () => {
 			const expectedPath = new URL(path, astroFixture.config.root).href;
 			assert.equal(writtenFiles[expectedPath].includes(content), true, error);
 		},
+		thenFileShouldBeValidTypescript(path) {
```

Comment:
I don't think I understand the naming of this. Did you mean `typesFileShouldBeValidTypeScript`?

Reply:
I was trying to keep consistency with the functions above 🙂